packages/slice-machine/lib/models/common/widgets/ContentRelationship/index.ts

```diff
@@ -60,4 +62,21 @@ export const ContentRelationshipWidget: Widget<Link, typeof schema> = {
   FormFields,
   CUSTOM_NAME: "ContentRelationship",
   Form,
+  prepareInitialValues: (initialValues) => {
```

Comment:
Should this be types ? (initial values)

Reply:
The type is defined in the interface here: https://github.com/prismicio/slice-machine/blob/5b6a8efaea00332316ed8ceff940f7da3fcd2a06/packages/slice-machine/lib/models/common/widgets/Widget.ts#L26

Is that what you mean?

Reply:
yes, but I would have expected something without an `any` if possible.

The interface can have any I guess but here it's a special use case with a special type.
Would it be hard to type it? Do you already have the type maybe?